test/e2e_node/mirror_pod_test.go

```diff
@@ -370,6 +372,19 @@ func createStaticPodUsingNfs(nfsIP string, nodeName string, cmd string, dir stri
 	return nil
 }
 
+// NFSv3 and v4 clients mount exported paths differently.  For NFSv4, there is a distinguished filesystem which is the
+// root of all exported filesystem. This is specified with fsid=root or fsid=0 both of which mean exactly the same thing.
+// In NFSv3 this root filesystem is mounted as '/'.  This function is an attempt to try to gloss over those differences
+// and make this test reliable for both versions.
+func mountPath() string {
+	cmd := exec.Command("lsmod")
```

Comment:
Even if nfsv4 module is loaded it doesn't mean that it's used. We may want to look at mtab/run mount, etc instead of making assumptions like this.

Reply:
I don't think it will display in mtab (unless there is some other NFS filesystem that is already mounted).  My logic was that the current code makes assumes NFSv3, this assumption may still be wrong in some cases but is less wrong in that the tests pass :)